net-libs/libvncserver/libvncserver-0.9.12-r1.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit cmake-utils
+
+MY_PN="LibVNCServer"
```

Comment:
we typically put that above `inherit`

Reply:
Well.. the result of my rough ebuilds calculation:
```
inherit_before_my: 4065
my_before_inherit: 441
```
=P

Reply:
And what's the point in declaring non-eclass variables before inherit?

Reply:
OK, I put `MY_PN` above `inherit`.

Reply:
@a17r, nope, we don't.